site/app/models/gradeable/GradeableUtils.php

```diff
@@ -91,4 +92,19 @@ public static function getAllGradeableListFromUserId(Core $core, User $user): ar
         $core->getConfig()->setCourseLoaded(false);
         return ["gradeables" => $gradeables, "graded_gradeables" => $graded_gradeables, "submit_btns" => $submit_btns];
     }
+
+    /**
+     * This function assumes that you are calling it where Core has a defined course
+     *
+     * @return \app\models\Button[][]|Gradeable[][]|GradedGradeable[][]|array[]
+     * @throws \Exception
```

Comment:
There's no particular need to add an `@throws` tag for a top-level `Exception`.  It's better to use it if there's a specific exception type potentially being thrown.